src/enterprise/consts.ts

```diff
@@ -1,4 +1,10 @@
 export const SELF_HOSTED_SERVER_CONFIGURATION = `tabnineSelfHostedUpdater.serverUrl`;
+
+export const SELF_HOSTED_IGNORE_PROXY_CONFIGURATION =
+  "tabnineSelfHostedUpdater.useProxySupport";
+
+export const IGNORE_PROXY_CONFIGURATION = "tabnine.useProxySupport";
```

Comment:
it should be part of the package.json

Reply:
it's not new, just in a const now because it is in use in few places